spi/src/main/java/org/eclipse/dataspaceconnector/spi/contract/offer/ContractDefinitionResult.java

```diff
@@ -13,6 +13,8 @@
  */
 package org.eclipse.dataspaceconnector.spi.contract.offer;
 
+import org.eclipse.dataspaceconnector.spi.types.domain.contract.offer.ContractDefinition;
+
 /**
  * The result of searching for a specific contract definition.
  */
```

Comment:
The entire class seems to be just an elaborate wrapper around a `null` check, so I think it can be removed.

Reply:
That's a good point, I'll remove it